tests/compile-fail/sync/libc_pthread_rwlock_read_write_deadlock_single_thread.rs

```diff
@@ -1,4 +1,5 @@
 // ignore-windows: No libc on Windows
+// error-pattern: deadlock
```

Comment:
these should now have better spans again, too, right?

Reply:
They could be, yeah -- but these tests don't care so much about the span so I figured "meh"^^